griptape/tasks/base_audio_generation_task.py

```diff
@@ -6,14 +6,14 @@
 from attrs import define
 
 from griptape.configs import Defaults
-from griptape.mixins import BlobArtifactFileOutputMixin, RuleMixin
+from griptape.mixins import ArtifactFileOutputMixin, RuleMixin
 from griptape.tasks import BaseTask
 
 logger = logging.getLogger(Defaults.logging_config.logger_name)
 
 
 @define
-class BaseAudioGenerationTask(BlobArtifactFileOutputMixin, RuleMixin, BaseTask, ABC):
+class BaseAudioGenerationTask(ArtifactFileOutputMixin, RuleMixin, BaseTask, ABC):
```

Comment:
out of scope for thisPR, but should we add some kind of optional persistence driver to `BaseTask` rather than have this mixin?

Reply:
Yes totally